spec/tictactoe_clojure/game_spec.clj

```diff
@@ -0,0 +1,14 @@
+(ns tictactoe-clojure.game-spec
+  (:require [speclj.core :refer :all]
+            [tictactoe-clojure.human-player :as human-player]
+            [tictactoe-clojure.console :as console]
+            [tictactoe-clojure.game :refer :all]))
+
+(describe "tictactoe-clojure.game"
+  (with new-board [1 2 3 4 5 6 7 8 9])
+  
+  (describe "#play-game"
+    (it "loops through a game where the human enters 1, 2, 4 and loses to the computer"
+      (with-redefs [human-player/human-move (fn [_] (with-in-str "1\n2\n4\n"))
+                    console/display (fn [_] "Player O wins")]
+          (should= "Player O wins" (play-game @new-board "X"))))))
```

Comment:
Let's say that your computer now has a different strategy to the game and does not play the same moves. I don't think this test will pass then. Now, this test relies on the computer making a certain set of moves and is therefore brittle. Please reach out to a crafter or find a time to pair with one of the mentors if you need help writing this